lua/treesitter-context/context.lua

```diff
@@ -215,6 +215,7 @@ local function get_parent_langtrees(bufnr, range)
     return {}
   end
 
+  root_tree:parse(range)
```

Comment:
Parsing can be async now, so this will have a perf hit.

Maybe we could try and pass an empty `on_parse` param.